File: Week 1/cash.c
#include<stdio.h>
#include<math.h>
#include<cs50.h>
int main(void)
{
    float dollars;
    int coins = 0;
    do
    {
        dollars = get_float("Change owed:");
    }
    while (dollars < 0);

    int cents = round(dollars * 100);
    //printf("%d",cents);
    while (cents != 0)
    {
        if (cents >= 25)
        {
            cents = cents - 25;
            coins++;
        }
        else if (cents >= 10)
        {
            cents = cents - 10;
            coins++;
        }
        else if (cents >= 5)
        {
            cents = cents - 5;
            coins++;
        }
        else if (cents >= 1)
        {
            cents = cents - 1;
            coins++;
        }


    }
    printf("%i\n", coins);


}

File: Week 6/cash/cash.py

from cs50 import get_float


dollars = get_float("Change owed: ")
coins = 0

cents = round(dollars * 100)
while cents != 0:
    if cents >= 25:
        cents = cents - 25
        coins += 1
    elif cents >= 10:
        cents = cents - 10
        coins += 1
    elif cents >= 5:
        cents = cents - 5
        coins += 1
    elif cents >= 1:
        cents = cents - 1
        coins += 1


print(coins)